pkgs/applications/networking/browsers/qutebrowser/default.nix

```diff
@@ -101,7 +101,8 @@ python3.pkgs.buildPythonApplication {
   in
     ''
     makeWrapperArgs+=(
-      "''${gappsWrapperArgs[@]}"
```

Comment:
Why isn't this part of the previous commit?

Reply:
looks like just a typo

Reply:
> looks like just a typo

Would be nice to fix this small git history thing, if we are already at it.